types/mongodb/index.d.ts

```diff
@@ -1407,13 +1407,25 @@ export interface FindOneAndReplaceOption extends CommonOptions {
     maxTimeMS?: number;
     upsert?: boolean;
     returnOriginal?: boolean;
+    collation?: CollationDocument;
 }
 
 /** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#findOneAndUpdate */
 export interface FindOneAndUpdateOption extends FindOneAndReplaceOption {
     arrayFilters?: Object[];
 }
 
+/** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#findOneAndDelete */
+export interface FindOneAndDeleteOption {
+    projection?: Object;
+    sort?: Object;
```

Comment:
Can you please change in `Object` in `object` so it will not error after https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33276

Thanks